Kubernetes/omsagent.yaml

```diff
@@ -11,7 +11,9 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["pods", "events", "nodes", "namespaces", "services"]
-  verbs: ["list"]
+  verbs: ["list", "get", "watch"]
+- nonResourceURLs: ["/metrics"]
```

Comment:
Does this include a leading / or does it not matter? (the nonResourceURLs)

Reply:
its a URL. it is required to have /metrics.